source/fuzz/transformation_add_parameter.cpp

```diff
@@ -85,6 +86,11 @@ void TransformationAddParameter::Apply(
   // TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/3403):
   //  Add an PointeeValueIsIrrelevant fact if the parameter is a pointer.
 
+  // Mark new parameter as irrelevant so that we can replace its use with some
+  // other id.
+  transformation_context->GetFactManager()->AddFactIdIsIrrelevant(
+      message_.parameter_fresh_id());
+
```

Comment:
Do you have test in a forthcoming PR that will showcase this behaviour working?

Reply:
Yes, we've decided to make a separate PR to adjust tests.